config/settings.yml

```diff
@@ -80,8 +80,6 @@
   :scan_via_host: true
   :use_vim_broker: true
   :use_vim_broker_ems: true
-:container_scanning:
```

Comment:
@moolitayer well technically anything that is container-generic (as vm-generic) it should be here probably (e.g. in the future maybe not all the container providers will be kubernetes-based).

Anything that is in kubernetes (and used in openshift as well) then it belongs to `manageiq-providers-kubernetes`.